src/patternfly/demos/Toolbar/examples/toolbar-complex-example.hbs

```diff
@@ -0,0 +1,99 @@
+{{#> toolbar pf-l-toolbar--modifier="pf-u-justify-content-space-between pf-u-mx-xl pf-u-my-md"}}
+    {{#> toolbar-section aria-label="First section"}}
+        {{#> toolbar-group}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-2xl"}}
+                {{#> input-group}}
+                    {{#> dropdown id="dropdown-example-collapsed"}}
+                    All
+                    <i class="fas fa-caret-down pf-c-dropdown__toggle-icon"></i> 
+                    {{/dropdown}}
+                    {{#> input-group-action input-group-action--modifier="pf-m-align-right" inputGroupTextType="button" aria-label="popover for input"}}
+                    <i class="fas fa-search aria-hidden="true"></i>
+                    {{/input-group-action}}
+                    {{#> form-controls controlType="input" input="true" type="text" id="textInput10" name="textInput10" placeholder="Filter..." aria-label="standard input example" }}
+                    {{/form-controls}}
+                {{/input-group}}
+            {{/toolbar-item}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-lg"}}
+                {{#> dropdown id="dropdown-example-collapsed" pf-c-dropdown--HasToggleIcon="true"}}
+                    All
+                {{/dropdown}}
+            {{/toolbar-item}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-m-sm pf-u-mr-lg"}}
+                <a class="pf-m-link"><i class="fas fa-sort-alpha-down"></i></a>
+            {{/toolbar-item}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-lg"}}
+                17 of 80 Items
+            {{/toolbar-item}}
+        {{/toolbar-group}}
+        {{#> toolbar-group}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-m-sm"}}
+                <a class="pf-m-link"><i class="fas fa-list"></i></a>
+            {{/toolbar-item}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-m-sm"}}
+                <a class="pf-m-link"><i class="fas fa-table"></i></a>
+            {{/toolbar-item}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-my-sm pf-u-mx-lg"}}
+                <a class="pf-m-link">Action</a>
```

Comment:
this should be a button as well. Once you make that change I think you'll need to adjust the spacers as well.

Reply:
fixed